doc/user_guide/conf.py

```diff
@@ -267,8 +276,10 @@
 
 # Bibliographic Dublin Core info.
 epub_title = u'PSyclone'
-epub_author = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter and Sergi Siso'
-epub_publisher = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter and Sergi Siso'
+epub_author = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter and ' \
+    'Sergi Siso'
+epub_publisher = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter ' \
+    'and Sergi Siso'
 epub_copyright = u'2017-2019, STFC'
```

Comment:
Please update the copyright dates in this file too.

Reply:
Done